### ISSUE TITLE
#each block ignored

### ISSUE THREAD

**Issue by GeeWizWow** [ericf/express-handlebars#156]
Hi 

I'm having an issue with the each block being ignored at render

as per the example here: http://handlebarsjs.com/builtin_helpers.html

```
res.render('home',{
            helpers: {
                people: [
                    "Yehuda Katz",
                    "Alan Johnson",
                    "Charles Jolley"
                ]
            }
        });

<ul>
<li>Test</li>
{{#each people}}
    <li>{{this}}</li>
{{/each}}
</ul>
```
Renders this:

```
<ul>
<li>Test</li>
</ul>
```

Using express-handlebars@2.0.1

Thanks for your time

**Comment by rauberdaniel** [comment]
you should either do `{{#each helpers.people}}` or move the `people` property outside the `helpers` object.

**Comment by jpike88** [comment]
This drove me almost INSANE trying to resolve, I spent too much time looking into handlebars documentation to then come back and find this issue. 

You should be having anything defined in 'helpers' bubble upwards automatically as this is not how handlebars normally works.

**Comment by chrisbutler** [comment]
yeah, this is ridiculously bad and not documented as far as i can tell...
